app/models/insured_eligible_for_benefit_rule.rb

```diff
@@ -65,6 +67,22 @@ def satisfied?
     [false]
   end
 
+  def eligibility_errors(element)
+    person = @role.person
+    pronoun = person.try(:gender) == 'male' ? ' he ' : ' she '
+    name = person.try(:first_name) || ''
+    case element
+    when "citizenship_status"
+      "Since #{name} did not attest to being a US citizen or having an eligible immigration status,#{pronoun}is not eligible to purchase a plan on #{Settings.site.short_name}.<br/> Other family members may still be eligible to enroll"
```

Comment:
We're not using `Settings` anymore, please use `EnrollRegistry[:enroll_app].setting(:short_name).item` https://github.com/ideacrew/enroll/blob/trunk/system/config/templates/features/enroll_app/enroll_app.yml